### DISCUSSION TITLE
Creating a subfolder

### DISCUSSION THREAD

**Discussion by rosharp** [massCodeIO/massCode#39] [Ideas]
I would like to create a 'React' subfolder in 'JS' folder in order to make it more concise. 

What are your thoughts on adding this feature?

**Comment by antonreshetov** [comment]
I don't quite understand the request. If you want to create a subfolder, then create a new folder and drag it to the desired folder.

**Reply by Genzo4** [reply]
This is not frendly interface. Maybe it would be better to make a button "Create subfolder"?
